src/components/ToolTree.tsx

```diff
@@ -3,70 +3,100 @@ import Img from 'next/image';
+      {Object.entries(tree).map(([key, value]) => (
+        <Disclosure key={key} defaultOpen={true}>
+          {({ open }) => (
+            <>
+              <Disclosure.Button className="mt-6 flex items-center justify-between">
+                <div className="flex gap-3">
+                  <Img src={value.icon} height={16} width={16} />
+                  <p className="uppercase text-deth-gray-300"> {key} </p>
+                </div>
+                {open ? (
+                  <Img
+                    className="text-red-100"
+                    src="/static/svg/minus.svg"
+                    width={18}
+                    height={18}
+                  />
+                ) : (
+                  <Img
+                    className="text-red-100"
+                    src="/static/svg/plus.svg"
+                    width={18}
+                    height={18}
+                  />
+                )}
+              </Disclosure.Button>
 
-                {(value as Tool[]).map((tool) => (
-                  <Disclosure.Panel
-                    key={tool.title}
-                    className="min-w-38 mt-3 flex flex-col items-start"
+              {value.tools.map((tool) => (
+                <Disclosure.Panel
+                  key={tool.title}
+                  className="min-w-38 mt-2 flex flex-col items-start"
+                >
+                  <MyLink
+                    href={`/${tool.pageHref}`}
+                    className="mr-auto flex h-10 items-center rounded-lg px-4 hover:bg-deth-gray-600 hover:text-white"
                   >
-                    <MyLink
-                      href={`/${tool.pageHref}`}
-                      className="mr-auto flex h-10 items-center rounded-lg px-4 hover:bg-black hover:text-white"
-                    >
-                      {tool.isNew && (
-                        <p className="mr-2 rounded-lg bg-purple-200 px-0.5">
-                          {' '}
-                          NEW{' '}
-                        </p>
-                      )}
-                      {tool.title}
-                    </MyLink>
-                  </Disclosure.Panel>
-                ))}
-              </>
-            )}
-          </Disclosure>
-        ))}
-      </div>
-    </section>
+                    {tool.isNew && (
+                      <p className="mr-4 animate-pulse rounded-lg bg-deth-purple px-2 font-semibold text-deth-white duration-700">
+                        NEW
+                      </p>
+                    )}
+                    <p className="text-deth-white">{tool.title}</p>
+                  </MyLink>
+                </Disclosure.Panel>
+              ))}
+            </>
+          )}
+        </Disclosure>
+      ))}
+    </div>
+  );
+}
+
+export function ToolTree({
+  className,
+  isShowMobileTree,
+  isMobile,
+}: {
+  className?: string;
+  isShowMobileTree: boolean;
+  isMobile: boolean;
+}) {
+  return (
+    <>
+      {isShowMobileTree && isMobile ? (
```

Comment:
I'd probably prefer media queries or tailwind responsive prefixes over `isMobile` computed in JavaScript, but I don't think it's so important we should change it.